src/leaflike/bookmarks/db.clj

```diff
@@ -1,15 +1,27 @@
 (ns leaflike.bookmarks.db
   (:require [clojure.java.jdbc :as jdbc]
-            [leaflike.config :refer [db-spec]]
-            [honeysql.core :as sql]
-            [honeysql.helpers :as helpers]))
+            [honeysql.core     :as sql]
+            [honeysql.helpers  :as helpers]
+            [honeysql.types    :as types]
+            [clojure.string    :as str]
+            [leaflike.config   :refer [db-spec]]))
+
+(defn format-tags
+  [{:keys [tags] :as params}]
+  ;; check if tag exists
+  ;; check if tags is nil?
+  (if (nil? tags)
+    params
+    (do ;; :tags "space space , comma . dot"
+        (str/split tags #"(\w)(\s+)([\.,])")
```

Comment:
The result of this split isn't being used anywhere. tags is still the unsplit string.

Also, why isn't (str/split tags #",") enough?